libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -737,11 +753,18 @@ void AC_Avoid::adjust_velocity_polygon(float kP, float accel_cmss, Vector2f &des
     // We need a separate vector in case adjustment fails,
     // e.g. if we are exactly on the boundary.
     Vector2f safe_vel(desired_vel_cms);
+    Vector2f safe_accel(desired_accel_cmss);
+
+    // Active avoidance distances
+    Vector2f max_limits;
```

Comment:
for me "max" and "limits" are pretty similar words.  Maybe it's distance_max?  not sure..